botbuilder-teams-js/package.json

```diff
@@ -12,9 +12,10 @@
   },
   "typings": "./lib/index.d.ts",
   "dependencies": {
-    "adaptivecards": "^1.0.0",
-    "botbuilder": "4.3.2",
-    "botframework-connector": "4.3.2"
+    "adaptivecards": "^1.0.0",    
+    "botbuilder": "^4.3.2",
+    "botframework-connector": "^4.3.2",
+    "jsdom": "14.0.0"
```

Comment:
jsdom has [type binding](https://www.npmjs.com/package/@types/jsdom). Please `npm i -D @types/jsdom` (don't use --save as we only need types as devDependencies)

Reply:
done